python/src/ExponentialFactory_doc.i.in

```diff
@@ -4,15 +4,25 @@
 Available constructor:
     ExponentialFactory()
 
-The parameters are estimated by method of moments:
+Let :math:`\bar{x}` be the sample mean. 
+Let :math:`x_{(1)}` be the sample minimum. 
+The parameter :math:`\gamma` is estimated by method of maximum 
+likelihood, with asymptotic correction for bias:
 
 .. math::
     :nowrap:
 
-    \begin{eqnarray*}
-      \displaystyle \Hat{\gamma}_n = x_{(1,n)} - \frac{|x_{(1,n)}|}{2+n}\\
-      \displaystyle \Hat{\lambda}_n= \frac{1}{\bar{x}_n-\Hat{\gamma}_n}
-    \end{eqnarray*}
+      \hat{\gamma} = x_{(1)} - \frac{|x_{(1)}|}{2 + n}
```

Comment:
I don't understand this estimator, though. If lambda is small or gamma is large, there could be a huge bias. 

The minimum of a sequence of iid exponential random variable X^{(1)} follows an exponential distribution with the same gamma but with lambda multiplied by n.

Assuming gamma to be nonnegative, the bias is asymptotically (1/lambda - gamma)/n

Reply:
The method of moments estimators of lambda and gamma are:
![image](https://user-images.githubusercontent.com/31989332/112591281-6e82a080-8e04-11eb-8413-21bcb7811983.png)
These estimators are unbiased.

Reply:
I am sorry, but I do not exactly know where this estimator comes from. I guess that this is a asymptotic correction of the bias of the MLE for the location parameter gamma. I was not, however, able to find a paper on this topic. @regislebrun : may you provide a reference on this topic?

Reply:
@josephmure : from what reference did you get these equations?

Reply:
I have no reference to give you, I simply did the calculation by hand. Let X_i (i=1,...,n) be iid random variables following the exponential distribution with parameters lambda and gamma. Let X^(1) be the minimum of the X_i (i=1,...,n). Then for any integer i between 1 and n:

![image](https://user-images.githubusercontent.com/31989332/114070011-e12d5a80-989f-11eb-8448-0ac8f7980e9a.png)

Moreover, this quantity can be estimated (without bias) by:

![image](https://user-images.githubusercontent.com/31989332/114070268-1d60bb00-98a0-11eb-890f-7d8e5a27d922.png)

This yields the estimator of lambda. The estimator of gamma comes from replacing lambda by its estimator in the expression of the expected value of X^(1), which is gamma + 1/n/lambda.

Reply:
A reference for the biais of the maximum likelihood estimator of parameters of the exponential distribution is:
"Introduction au calcul des probabilités et à la statistique.". Jean-François Delmas. Les Presses de l’ENSTA, 2010. Exemple VIII.43 page 225.

Reply:
A few additional remarks:

![image](https://user-images.githubusercontent.com/31989332/127997995-433087b2-4dd6-4a15-84b7-004feb197075.png)


follows the Gamma(n-1, (n-1) lambda) distribution. We can deduce from this that its variance is (n-1)^(-1) lambda^(-2).